lib/modules/manager/terraform/__fixtures__/modules.tf.json

```diff
@@ -0,0 +1,129 @@
+{
```

Comment:
reduce fixtures to minimum size

Reply:
I initially resolved this by removing some coverage. But I do want to ask, why do we want to reduce the coverage which would make sure the behavior between the `.tf` and the `.tf.json` remains the same?